### ISSUE TITLE
Cannot find module `bluebird`

### ISSUE THREAD

**Issue by ehmicky** [Munter/netlify-plugin-subfont#33]
When loading this plugin, some builds are showing this error:

```
Error: Could not import plugin
Cannot find module 'bluebird'
```

This is coming from the first line of `assetgraph/lib/AssetGraph.js`, which after bundling is:

```js
const Promise = require('bluebird');
```

I am not quite sure why `bluebird` is not installed considering this is a dependency of `assetgraph`. @Munter Do you have some idea on what might have happened?

**Comment by Munter** [comment]
The best I can come up with is some quirk in either the npm installation or the cross-build caching mechanism in netlify builds. I'm not doing anything out of the ordinary while installing assetgraph and neither is assetgraph while installing bluebird

**Comment by ehmicky** [comment]
Yes this is possible. I'll monitor it and see if the problem arises again.

**Comment by Munter** [comment]
This might actually be related to #28
